src/NUnitConsole/nunit3-console/ConsoleRunner.cs

```diff
@@ -238,12 +244,20 @@ private int RunTests(TestPackage package, TestFilter filter)
                 {
                     var outputPath = Path.Combine(_workDirectory, spec.OutputPath);
                     GetResultWriter(spec).WriteResultFile(result, outputPath);
-                    _outWriter.WriteLine("Results ({0}) saved as {1}", spec.Format, spec.OutputPath);
+                    writer.WriteLine("Results ({0}) saved as {1}", spec.Format, spec.OutputPath);
```

Comment:
This change is not related to the main PR, but believe this should use the 'new' writer instead. See discussion here: https://github.com/nunit/nunit-console/pull/320#discussion_r154501112 - came across this after my own confusion!